src/workers/notifications/postAdded.ts

```diff
@@ -66,6 +74,23 @@ const worker: NotificationWorker = {
             } as NotificationPostContext & Partial<NotificationDoneByContext>,
           }),
         );
+
+        const posts = await con.getRepository(SharePost).countBy({
+          authorId: post.authorId,
+        });
+        if (posts === 1) {
+          const subscribed = await con.getRepository(UserAction).findOneBy({
+            userId: post.authorId,
+            type: UserActionType.EnableNotification,
```

Comment:
Actually should be `squad_first_post` according to the specs right?
<img width="776" alt="Screenshot 2023-05-02 at 13 08 19" src="https://user-images.githubusercontent.com/554874/235650290-cfe0db14-89ea-4b1f-94ba-5fdd63d0ab5b.png">

Reply:
Oh wait think you wanting to check if the user is already subscribed.
Got it, but then we also need to validate they don't already have this right? (Case of secondary squad for instance)

Reply:
Yes, this is about whether we should still send a notification for them to subscribe.

> Got it, but then we also need to validate they don't already have this right? (Case of secondary squad for instance)

I think it should not be necessary now since our condition was just `authorId` without checking whichever Squad so this can only happen once. Unless the user deleted their post and then share something again, that is an edge case. Should we cover it or it is highly unlikely?

Reply:
> squad_first_post

They are kinda relative to one another, so it is really a bit confusing. In the task I have for migration and retro checks, it will cover this case.